src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/ValidationRulePart.Fields.Edit.cshtml

```diff
@@ -0,0 +1,27 @@
+@using OrchardCore.Forms.ViewModels;
+@model ValidationRulePartEditViewModel
+
+<div class="form-group" >
+    <label asp-for="Type">@T["Validate Type"]</label>
```

Comment:
The expression is more like "validation" than "validate" so please change it in every text.